macOS/DuckDuckGo/Common/Localizables/UserText.swift

```diff
@@ -901,8 +901,11 @@ struct UserText {
     static let onStartup = NSLocalizedString("preferences.on-startup", value: "On Startup", comment: "Name of the preferences section related to app startup")
     static let reopenAllWindowsFromLastSession = NSLocalizedString("preferences.reopen-windows", value: "Reopen all windows from last session", comment: "Option to control session restoration")
     static let showHomePage = NSLocalizedString("preferences.show-home", value: "Open a new window", comment: "Option to control session startup")
+    static let openANew = NSLocalizedString("preferences.startup.open-a-new", value: "Open a new:", comment: "Label for startup window type selection")
+    static let window = NSLocalizedString("preferences.startup.window", value: "Window", comment: "Option for regular window type")
     static let fireWindow = NSLocalizedString("preferences.fire-window-section", value: "Fire Window", comment: "Section title for Fire Window preferences")
     static let openFireWindowByDefault = NSLocalizedString("preferences.open-fire-window-by-default", value: "Open Fire Window by default", comment: "Option to make all new windows Fire Windows")
+    static let openFireWindowByDefaultExplanation = NSLocalizedString("preferences.open-fire-window-by-default.explanation", value: "Open Fire Window by default when opening a new window (Command + N). You can still open a regular browser window when this setting is on with Shift + Command + N.", comment: "Explanation for the option to make all new windows Fire Windows")
```

Comment:
Shouldn’t we use ⌘N and ⇧⌘N in the copy? I’d personally prefer that.

Reply:
Yes, good call. I’m making some changes to the Copy (after some Design review). I will update and propose to use the emoji/icons instead.

Reply:
Thanks! Ideally we should parametrize the shortcuts. We do that for the history onboarding message:
```swift
UserText.historyViewOnboardingMessage(shortcut: "⌘Y”)
```